driver/pparse.ml

```diff
@@ -166,6 +166,9 @@ let file ppf ~tool_name inputfile parse_fun ast_magic =
   close_in ic;
   apply_rewriters ~restore:false ~tool_name ast_magic ast
 
+let file ppf ~tool_name inputfile parse_fun ast_magic =
+  Timings.(time (Parsing inputfile))
+    (file ppf ~tool_name inputfile parse_fun) ast_magic
```

Comment:
To separate the timings of parsing an preprocessing I needed to keep it here (on parse_implementation and parse_interface).